pkgs/tools/filesystems/bcachefs-tools/Makefile.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile b/Makefile
+index af7a206..553ac70 100644
+--- a/Makefile
++++ b/Makefile
+@@ -47,7 +47,7 @@ ifeq ($(PREFIX),/usr)
+ 	INITRAMFS_DIR=$(PREFIX)/share/initramfs-tools
+ else
+ 	ROOT_SBINDIR=$(PREFIX)/sbin
+-	INITRAMFS_DIR=/etc/initramfs-tools
++	INITRAMFS_DIR=$(PREFIX)/etc/initramfs-tools
```

Comment:
What is this for? Is this configuration really meant to be read-only?

Reply:
It wouldn't finish the `make install` command without it.
The path there is absolute, and assumed a normal linux installation, not something like NixOS.

Reply:
I'm guessing it's trying to create that path, which is of course bad. I wonder why, we should open an issue.

It is odd, since he even has a branch with nix support :) https://evilpiepirate.org/git/bcachefs-tools.git/commit/?h=dev

In the meantime, I'll merge, thanks!